index.html

```diff
@@ -4,27 +4,35 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
     <meta name="description" content="Kashav Madan, Toronto-based software engineer." />
-    <title>Kashav Madan</title>
-    <link href="https://fonts.googleapis.com/css?family=Inconsolata:400" rel="stylesheet" type="text/css" />
+    <title>Kashav Madman</title>
+    <link href="https://fonts.googleapis.com/css?family=Major+Mono+Display" rel="stylesheet" type="text/css" />
     <style type="text/css">
       * { margin: 0; padding: 0; }
       html { width: 100%; height: 100%; }
       body { padding: 20px;
              font-size: 0.9rem;
-             font-family: "Inconsolata";
+             font-family: 'Major Mono Display', monospace;
              line-height: 1rem; }
       p { color: rgb(105, 105, 105);
-          max-width: 200px;
+          max-width: 280px;
           word-wrap: break-word; }
       a { color: rgb(5, 5, 5); }
       a:hover { text-decoration: none;
                 background-color: lavenderblush; }
+      .main-container {
+        width: 100%;
+        text-align: end;
+        display: grid;
+        justify-items: end;
+      }
     </style>
   </head>
   <body>
-    <p>engineer on the <a href="https://wiki.mozilla.org/Project_Fission">fission</a> team at mozilla</p>
-    <br />
-    <p>github: <a href="https://github.com/kashav">kashav</a>, <a href="https://github.com/kshvmdn">kshvmdn</a></p>
-    <p>contact: <a href="mailto:kshvmdn@gmail.com">kshvmdn@gmail.com</a><p>
+    <div class="main-container">
```

Comment:
I'm trying to keep the environmental footprint of this project as low as possible, so I'd really prefer if we didn't introduce a separate class for this. It's the only `<div>` in the file, we can just select it directly with `div`.